packages/rln/src/zerokit.ts

```diff
@@ -65,23 +66,31 @@ export class Zerokit {
     uint8Msg: Uint8Array,
     memIndex: number,
     epoch: Uint8Array,
-    idKey: Uint8Array
+    idKey: Uint8Array,
+    rateLimit: number = DEFAULT_RATE_LIMIT
```

Comment:
since `ZeroKit` is standalone stateful class - we should seed it with `rateLimit` config upon creation and avoid setting it on each `generate`/`validate` etc

any reason not to, @danisharora099 ?

Reply:
fair point, no reason; updated!